core/analysis/analyzers.cpp

```diff
@@ -103,22 +103,22 @@ struct hash<::key> {
 
 namespace {
 
-const std::string FILENAME_PREFIX("libanalyzer-");
+const std::string kFileNamePrefix("libanalyzer-");
```

Comment:
NIT
```suggestion
constexpr std::string_view kFileNamePrefix = "libanalyzer-";
```

Reply:
Code `std::string{some_string_view}.c_str()` is necessary as string_view is not guaranteed to be null terminated

Reply:
I know, but I think in practice we use string_view constructed from string

Reply:
I hope we can improve that once `std::format(...)` is available.